blocks/top-table-list-block/features/top-table-list/_children/item-title-with-right-image.test.jsx

```diff
@@ -84,12 +84,12 @@ describe('item title with right image block', () => {
       />,
     );
 
-    // expect(wrapper.find('h2.simple-list-headline-text').length).toBe(1);
+    expect(wrapper.find('h2.simple-list-headline-text').length).toBe(0);
```

Comment:
no text found? I would expec this to still be 1 headline found?

Reply:
I'm not sure what's going on with this, but as Amy's currently working on the small item she asked me not to make changes to it.